configuration/outputs.md

```diff
@@ -360,8 +360,9 @@ If enabled, the SignalFx integration will stream analytics and insights to an Si
 
 ## Trigger Destinations
 
-### **Slack**
+### **Slack/Webhook**
 
+Slack and webhook configuration are the same so you can follow slack to add a webhook trigger.<br>
```

Comment:
webhook and slack are not exactly same. webhook needs its own section to mention below additional fields
- username/password
- custom_headers
- endpoint

Also add examples on how to configure pagerduty and edgedelta email service